flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -1932,6 +1942,56 @@ class TableEnvironmentTest {
           "Please instantiate a new TableEnvironment if necessary.")
   }
 
+  @Test
+  def testAddPartitions(): Unit = {
```

Comment:
Shoud rename/drop/show partition also do the same test?
https://github.com/apache/flink/pull/19553
https://github.com/apache/flink/pull/19544
https://github.com/apache/flink/pull/19545